app/main.py

```diff
@@ -1,3 +1,19 @@
 def create_dict(*args):
-    # write your code here
-    pass
+    dicto = {}
+    for index, arg in enumerate(args):
+        if isinstance(arg, tuple):
```

Comment:
FAILED tests/test_main.py::test_create_dict[args7-Cannot add (1, [2, 3]) to the dict!\n-expected_result7]